code/game/objects/items/weapons/storage/backpack.dm

```diff
@@ -625,4 +619,75 @@
 	desc = "A small, fashionable bag typically worn over the shoulder."
 	icon_state = "purse"
 	item_state = "purse"
-	max_storage_space = 16
\ No newline at end of file
+	max_storage_space = 16
+
+//**Vaurca cloaks**//
+
+/obj/item/storage/backpack/cloak
+	name = "tunnel cloak"
+	desc = "A Vaurca cloak with storage pockets."
+	icon_state = "cape"
+	item_state = "cape"
+	w_class = ITEMSIZE_HUGE
+	slot_flags = SLOT_BACK
+	max_w_class = ITEMSIZE_NORMAL
+	max_storage_space = 28
```

Comment:
If you have the same variable value at the child and the parent you should not explicitly assign it at the child.